deploy/olm-catalog/akka-cluster-operator/0.2.1/akka-cluster-operator-certified.v0.2.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator-certified.v0.2.1
+  namespace: placeholder
```

Comment:
placeholder ?

Reply:
The namespace listed in the CSV within a catalog is actually a placeholder, so it is common to simply list [placeholder.](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#csv-metadata)